db/migrations/20201028151313_card_game.js

```diff
@@ -6,7 +6,8 @@ exports.up = function (knex) {
       table.text('instructions', 10000);
       table.text('description', 10000);
       table.string('materials');
-      table.string('number_of_players');
+      table.string('min_players');
+      table.string('max_players');
```

Comment:
This change makes it easy to read and to work with. I am glad you made the change.